main.go

```diff
@@ -114,11 +116,12 @@ func spin() {
 func doPeriodicMode(es *elasticsearch.Elasticer, d *database.Databaser, client *messaging.Client) {
 	logcabin.Info.Println("Periodic indexing mode selected.")
 
+	queueName := fmt.Sprintf("%s.periodic", amqpQueuePrefix)
```

Comment:
So I guess this sort of contradicts what I said above about the defaults, but it might be good to keep `templeton` in the static non-configurable parts of the string. Lots of stuff is periodic or incremental, but only this is (e.g.) both templeton and periodic. If we want to retain the current queue names for now with a default config, maybe the thing to do would be:

* add a default for the prefix, defaulting it to `""` (the empty string)
* when a non-empty-string prefix is set in config, the queue name is `sprintf("%s.templeton.periodic", prefix)`
* when the prefix is the empty string, the queue name is just "templeton.periodic" (i.e., no dot before 'templeton')

(and, of course, the same idea for the incremental queue name below)